app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,37 @@
+class PasswordResetsController < ApplicationController
+  layout 'static_pages'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.